packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualInterface.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+
+/**
+ * @notice External methods that the FundingRateStore needs access to.
+ */
+interface PerpetualInterface {
```

Comment:
New interface for the Perpetual which is now expected to be called by the FundingRateStore to determine the `collateralCurrency` and `fundingRateIdentifier` of the calling contract

Reply:
> Also, would it make sense to put this in an interfaces directory?

I can do this yeah but this would be the only `interface` in this directory so I held off for now

Reply:
> Are there other methods we'd want to expose here? we currently don't have an interface for the ExpiringMultiParty so exposing an interface like this is a slightly different pattern.

This interface is only used (currently) by the FundingRateStore, because the store uses `address perpetual` as function arguments, and then uses that to look up `perpetual.collateralCurrency()` or `perpetual.fundingRateIdentifier`. So this interface is specifically only useful to the Store and I kept its interface minimal (and referenced this in the comment)